tests/mentorship_relation/test_api_cancel_relation.py

```diff
@@ -64,16 +77,26 @@ def test__mentee_cancel_mentorship_relation(self):
         self.assertEqual(
             MentorshipRelationState.ACCEPTED, self.mentorship_relation.state
         )
+        self.assertIsNone(self.mentorship_relation.cancellation_reason)
+
         with self.client:
             response = self.client.put(
                 "/mentorship_relation/%s/cancel" % self.mentorship_relation.id,
                 headers=get_test_request_header(self.second_user.id),
+                content_type="application/json",
+                data=json.dumps(
+                    dict(cancellation_reason=self.cancellation_reason_example)
```

Comment:
Using literal {} syntax instead of dict() to create a dict is recommended and performs better

Reply:
@isabelcosta I think this calls for a new issue, to refractor the codebase and replace all dict() with {} for optimisation. I have updatedthe PR with this change, but maybe we need to maintain this consistency in our codebase.

Reply:
Thank you for updating this PR. And yes, this can be created into an issue :)